mixer/adapter/cloudwatch/README.md

```diff
@@ -0,0 +1,56 @@
+# Amazon CloudWatch Adapter
```

Comment:
Can you move all documentation into config.proto?

We harvest config.proto to put as adapter documentation on istio.io, whereas the README file would languish in GitHub unseen by our customers.